contracts/0.4.24/Lido.sol

```diff
@@ -50,6 +50,7 @@ contract Lido is ILido, IsContract, StETH, AragonApp {
 
     /// ACL
     bytes32 constant public PAUSE_ROLE = keccak256("PAUSE_ROLE");
+    bytes32 constant public SUBMITS_BREAK_ROLE = keccak256("SUBMITS_BREAK_ROLE");
```

Comment:
I prefer solid naming convention is stopSumbits -> SUBMITS_STOPPING_ROLE
Maybe its better to call it pause instead.

Reply:
Yep, fixed this one